src/_guides/language/effective-dart/design.md

```diff
@@ -476,6 +476,25 @@ can delete dead code. It can't do that if the member is public because it
 doesn't know if any code outside of its view is using it.
 
 
+### CONSIDER declaring multiple classes in the same library.
```

Comment:
How about:

> ### CONSIDER declaring _related_ classes in the same library.

Reply:
Actually, I was having a hard time thinking about an alternative title for this guideline. The content of this guideline seems to be more informative (along the lines of "by the way, you _can_ do this ...") than anything else. I suppose that one good reason to put more than one class in a library is because they are related. Another reason might be simply because the library is small (but then, I suppose, that the classes would then _still_ be related -- coincidental cohesion). Anyhow, kinda thinking out loud.

Reply:
Yeah, the guideline is a little weird. It technically carries no actual prescriptive content. Anything *not* prohibited by a guideline is permitted by default, so this is telling you nothing. But I've had a number of users believe there is some implicit that you *cannot* have more than one class per file — and assert that to other users in code reviews! — so I thought it worth adding a guideline to make this explicit.